jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -33,7 +33,12 @@ class Catalogs(PluginTemplateMixin, ViewerSelectMixin, HasFileImportSelect, Tabl
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.close_in_tray`
+    * :meth:`import_catalog`
     * :meth:`zoom_to_selected`
+    * :meth:`search`
+    * :attr:`results_available`
+    * :attr:`number_of_results`
```

Comment:
```suggestion
    * ``table`` (:class:`~jdaviz.core.template_mixin.Table`):
      Table containing all search results.
    * ``table_selected`` (:class:`~jdaviz.core.template_mixin.Table`):
      Table containing all selected search results.    
```

Reply:
does this not need to start with :attr: ?

Reply:
I don't think so, but this was just from memory, so let's definitely double check that the docs build correctly